### PR TITLE
Add fengari-loader to list of loaders

### PR DESCRIPTION
**Pull request by rhys-vdw**
I also changed the section "Transpilers" to "Dialects" because Fengari is a VM rather than a transpiler. Let me know if it would be preferable to create a new section instead.

### CODE REVIEW

**Comment by rhys-vdw** [comment]
Note that [list of loaders](https://github.com/webpack/docs/wiki/list-of-loaders) uses the heading "dialects"

File: src/content/loaders/index.md [hunk 1]
```diff
@@ -26,14 +27,15 @@ Loaders are activated by using `loadername!` prefixes in `require()` statements,
 * `cson-loader` Loads and transpiles a [CSON](https://github.com/bevry/cson#what-is-cson) file
 
 
-## Transpiling
+## Dialects
```
Comment (simon04) [inline src/content/loaders/index.md]: I'd consider "[transpiling](https://en.wikipedia.org/wiki/Source-to-source_compiler)" more concise than "dialect" and stick with this term.

File: src/content/loaders/index.md [hunk 2]
```diff
@@ -26,14 +27,15 @@ Loaders are activated by using `loadername!` prefixes in `require()` statements,
...
 
 * [`script-loader`](/loaders/script-loader) Executes a JavaScript file once in global context (like in script tag), requires are not parsed
 * [`babel-loader`](/loaders/babel-loader) Loads ES2015+ code and transpiles to ES5 using [Babel](https://babeljs.io/)
 * `buble-loader` Loads ES2015+ code and transpiles to ES5 using [Bublé](https://buble.surge.sh/guide/)
 * `traceur-loader` Loads ES2015+ code and transpiles to ES5 using [Traceur](https://github.com/google/traceur-compiler#readme)
 * [`ts-loader`](https://github.com/TypeStrong/ts-loader) or [`awesome-typescript-loader`](https://github.com/s-panferov/awesome-typescript-loader) Loads [TypeScript](https://www.typescriptlang.org/) 2.0+ like JavaScript
 * [`coffee-loader`](/loaders/coffee-loader) Loads [CoffeeScript](http://coffeescript.org/) like JavaScript
+* [`fengari-loader`](https://github.com/fengari-lua/fengari-loader/) Runs Lua code using [fengari](https://fengari.io/)
```
Comment (simon04) [inline src/content/loaders/index.md]: Runs → Loads

**Comment by simon04** [comment]
NB: Oh, since I migrated the list of loaders from the Wiki to this repository, https://github.com/webpack/webpack.js.org/commit/730e94f219d0d054b7b6b965f3473a5c159fc219, quite [some changes have been applied to the Wiki page](https://github.com/webpack/docs/wiki/list-of-loaders/_compare/ddefa66d81b079ee5734131d55b7a94a2013da2f...0622268f79e8c7a3363cbbe268147f418342b510). Maybe some parts should be integrated in our loader docs and/or in [awesome-webpack](https://github.com/webpack-contrib/awesome-webpack).

**Comment by rhys-vdw** [comment]
Added corrections and rebased onto master.

**Comment by montogeek** [comment]
Thanks!

**Pull request merged**